pulp_rpm/app/serializers.py

```diff
@@ -546,7 +547,7 @@ class Meta:
         fields = SingleArtifactContentUploadSerializer.Meta.fields + (
             'id', 'updated_date', 'description', 'issued_date',
             'fromstr', 'status', 'title', 'summary', 'version',
-            'type', 'severity', 'solution', 'release', 'rights',
+            'type', 'sha256', 'severity', 'solution', 'release', 'rights',
```

Comment:
Advisories are artifactless, I don't think this filter is needed here. @daviddavis , @lubosmj, what do you think?

Reply:
Well, if a class inherits from the class SingleArtifactContentUploadSerializer which inherits from the class SingleArtifactContentSerializer, I assume it may contain an artifact. @goosemania , what do you think?

Reply:
That's logical :) Unfortunately, advisories are always a special case.
It inherits from `SingleArtifactContentUploadSerializer` because a file is used to upload an advisory to Pulp.
Data from the file is parsed and saved into the DB, no artifact is saved in pulp storage or attached to that content unit.
To be more precise, during upload, the artifact is saved and then removed after being processed because it can't be a part of the advisory content unit.

Reply:
Alright then, what else is a special case? Only advisories?